addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/InsecureFormLoadScanRule.java

```diff
@@ -71,7 +71,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         }
     }
 
-    private boolean isHttps(HttpMessage msg) {
+    private static boolean isHttps(HttpMessage msg) {
```

Comment:
These unrelated changes should have been done in its own PR or at the very least in its own commit.

Reply:
I'll revert them, then do a more complete set in a separate PR.